### PR TITLE
added the ortho boards from the technik gb

### PR DESCRIPTION
**Pull request by daysgobye**
## Description

Added ortho version of technik keyboard the technik-o and the macro pad to go with it the beiwagon to the Boardsource folder.
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/boardsource/beiwagon/readme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Beiwagon
+
+![Beiwagon](https://imgur.com/eQi2Laq)
+
+*A short description of the keyboard/project*
```
Comment (drashna) [inline keyboards/boardsource/beiwagon/readme.md:5]: description here?

File: keyboards/boardsource/beiwagon/rules.mk [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+RGB_MATRIX_ENABLE = yes
+RGB_MATRIX_DRIVER = WS2812
+EXTRAFLAGS += -flto
```
Comment (drashna) [inline keyboards/boardsource/beiwagon/rules.mk:25]:
```suggestion
LTO_ENABLE = yes
```

File: keyboards/boardsource/technik_o/readme.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Technik-O
+
+![Technik-O](https://imgur.com/F2VTqAs)
+
+*A short description of the keyboard/project*
```
Comment (drashna) [inline keyboards/boardsource/technik_o/readme.md:5]: here too

File: keyboards/boardsource/technik_o/rules.mk [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+RGB_MATRIX_ENABLE = yes
+RGB_MATRIX_DRIVER = WS2812
+EXTRAFLAGS += -flto
```
Comment (drashna) [inline keyboards/boardsource/technik_o/rules.mk:25]:
```suggestion
LTO_ENABLE = yes
```

**Review by drashna — CHANGES_REQUESTED** [review]
You have changes to chibios libraries. 

You may need to rebase to fix these.

**Review by drashna — APPROVED** [review]

File: keyboards/boardsource/beiwagon/readme.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Beiwagon
+
+![Beiwagon](https://imgur.com/eQi2Laq)
```
Comment (fauxpark) [inline keyboards/boardsource/beiwagon/readme.md:3]:
```suggestion
![Beiwagon](https://i.imgur.com/eQi2Laqh.jpeg)
```

File: keyboards/boardsource/beiwagon/readme.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+    make boardsource/beiwagon:default
+
+Flashing example for this keyboard:
+
+    Reset keyboard by pushing reset button on back of pcb through access hole. Or hold down top left key while you plug in the usb cable.
+    Then run this command in your local qmk repo, or flash using qmk toolbox.
+    make boardsource/technik_s:default:flash
+
+    make boardsource/beiwagon:default:flash
```
Comment (fauxpark) [inline keyboards/boardsource/beiwagon/readme.md:21]:
```suggestion
    make boardsource/beiwagon:default:flash

Reset keyboard by pushing reset button on back of pcb through access hole. Or hold down top left key while you plug in the usb cable.
```

File: keyboards/boardsource/technik_o/readme.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+# Technik-O
+
+![Technik-O](https://imgur.com/F2VTqAs)
```
Comment (fauxpark) [inline keyboards/boardsource/technik_o/readme.md:3]:
```suggestion
![Technik-O](https://i.imgur.com/F2VTqAs.jpeg)
```

File: keyboards/boardsource/technik_o/readme.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+    make boardsource/technik_o:default
+
+Flashing example for this keyboard:
+
+    Reset keyboard by pushing reset button on back of pcb through access hole. Or hold down top left key while you plug in the usb cable.
+    Then run this command in your local qmk repo, or flash using qmk toolbox.
+    make boardsource/technik_s:default:flash
+
+    make boardsource/technik_o:default:flash
```
Comment (fauxpark) [inline keyboards/boardsource/technik_o/readme.md:21]:
```suggestion
    make boardsource/technik_o:default:flash

Reset keyboard by pushing reset button on back of pcb through access hole. Or hold down top left key while you plug in the usb cable.
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**